pkg/api/mlflow/api/response/metric.go

```diff
@@ -24,11 +24,36 @@ func NewMetricHistoryResponse(metrics []database.Metric) *GetMetricHistoryRespon
 
 	for n, m := range metrics {
 		response.Metrics[n] = MetricPartialResponse{
-			RunID:     m.RunID,
 			Key:       m.Key,
+			Step:      m.Step,
 			Value:     m.Value,
 			Timestamp: m.Timestamp,
+		}
+		if m.IsNan {
+			response.Metrics[n].Value = "NaN"
+		}
+	}
+	return &response
+}
+
+// GetMetricHistoryBulkResponse is a response object for `GET mlflow/metrics/get-history-bulk` endpoint.
+type GetMetricHistoryBulkResponse struct {
+	Metrics []MetricPartialResponse `json:"metrics"`
+}
+
+// NewMetricHistoryBulkResponse creates new GetMetricHistoryBulkResponse object.
+func NewMetricHistoryBulkResponse(metrics []database.Metric) *GetMetricHistoryBulkResponse {
+	response := GetMetricHistoryBulkResponse{
+		Metrics: make([]MetricPartialResponse, len(metrics)),
+	}
+
+	for n, m := range metrics {
+		response.Metrics[n] = MetricPartialResponse{
+			Key:       m.Key,
 			Step:      m.Step,
+			Value:     m.Value,
+			RunID:     m.RunID,
```

Comment:
nitpick: can you move up `RunID` to the top of the struct?

Reply:
ooook